.pre-commit-config.yaml

```diff
@@ -52,12 +45,6 @@ repos:
     hooks:
       - id: black-nb
 
-  # Enable this in repositories with python packages.
-  # - repo: https://github.com/mgedmin/check-manifest
-  #   rev: '0.39'
-  #   hooks:
-  #     - id: check-manifest
-
```

Comment:
You can completely remove this, the project will never be a package.

Reply:
isn't the file required by the aicoe to run pre-commit?

Reply:
That was meant for the commented-out parts - they can be removed from the file.